Source/FolioReaderPage.swift

```diff
@@ -656,7 +663,11 @@ extension UIWebView {
     // MARK: - Create and show menu
     
     func createMenu(options options: Bool) {
-        isShare = options
+		guard readerConfig.useReaderMenuController else {
+			return
```

Comment:
The thing is that `createMenu` is called from 6 diferent places but 3 of them only happen after an interaction with Folio menu items. That's why I prefered to put the check in the method itself instead of all 3 places. So should I add the check to these places as well or keep it in the method?